package.json

```diff
@@ -17,6 +17,7 @@
 		"@react-native-community/cli": "^12.1.1",
 		"@typescript-eslint/eslint-plugin": "^6.4.1",
 		"@wordpress/babel-preset-default": "file:gutenberg/packages/babel-preset-default",
+		"@wordpress/babel-plugin-import-jsx-pragma": "file:gutenberg/packages/babel-plugin-import-jsx-pragma",
```

Comment:
This addition is related to https://github.com/WordPress/gutenberg/pull/61692 where mobile still uses it.